site/src/pages/news/20200514-newsletter-0.mdx

```diff
@@ -0,0 +1,79 @@
+# Armeria Newsletter vol. 0
+
+14th May 2020
+
+<video class="hideOnReducedMotion" src="../../images/armeria.m4v" preload="none" autoplay muted loop style="width: 282px; height: 112px;">
+  <img src="../../images/armeria.gif" loading="lazy" width="282" height="112" />
+</video>
+
+## From the devs
+
+- [Our brand new web site](https://line.github.io/armeria) is now available! 🦄
+  - [Official documentation](https://line.github.io/armeria/docs) that looks better
+  - [Fascinating articles and slides](https://line.github.io/armeria/community/articles) from our community
+  - [Contributor manual](https://line.github.io/armeria/community) for your next participation
+- [Armeria 0.99.5](/release-notes/0.99.5) has been released.
+  - New <type://RetryRule> API that allows convenient construction of complex retry policy
+  - [Eureka service discovery](https://spring.io/guides/gs/service-registration-and-discovery/) support
+  - [`grpc-kotlin`](https://github.com/grpc/grpc-kotlin) support
+
+## From the community
+
+- [Monitoring a Spring Boot app in Kubernetes – What I learned from Devoxx Belgium 2019](https://engineering.linecorp.com/en/blog/monitoring-a-spring-boot-app-in-kubernetes-what-i-learned-from-devoxx-belgium-2019/)
+  - @nirvanarsc shows how he ran his Armeria-based Spring Boot application using Docker and Kubernetes.
+  - Learn about the basics of Reactive Streams, its industry adoption and interoperability.
+- [Lessons learned from the adoption of Armeria to LINE's authentication system](https://speakerdeck.com/line_developers/lesson-learned-from-the-adoption-of-armeria-to-lines-authentication-system?slide=8)
+  - @imasahiro tells about his successful migration from a home-grown HTTP/2 server based on Netty 4.1.0.Beta8 (!) to Armeria.
+
+## We need your comments
+
+Your opinion means a lot to us. Please let us know what you think about these proposals:
+
+- #2148 Provide a way to build an HTTP client request flently
```

Comment:
did you mean 'fluently'?

Reply:
Doh! Pushed the fix via 498d2dca530842bd070cb567ceffd07e4b256a98